### PR TITLE
fix(eredis): ssl_closed fix

### LINKED ISSUE #6784: Getting Generic server SSL error in emqx
We are getting below error in emqx logs every 5-10 seconds.

[error] Generic server <0.2030.0> terminating. Reason: {unhandled_message,{ssl_closed,{sslsocket,{gen_tcp,#Port<0.27>,tls_connection,undefined},[<0.2032.0>,<0.2031.0>]}}}. Last message: {ssl_closed,{sslsocket,{gen_tcp,#Port<0.27>,tls_connection,undefined},[<0.2032.0>,<0.2031.0>]}}. State: {state,"<redis host>",6380,<<"redis password">>,<<"0">>,undefined,3000,5000,{sslsocket,{gen_tcp,#Port<0.27>,tls_connection,undefined},[<0.2032.0>,<0.2031.0>]},{pstate,undefined,undefined},{[],[]}}.

Version of emqx: 4.3.10
Redis-- Azure Cache redis(With cluster enabled)

Note: we are using emqx_auth_redis to authenticate the clients

### PR DESCRIPTION
**Pull request by gsychev**
Fixed ssl_closed issue
Updated certificates

Fixes: https://github.com/emqx/emqx/issues/6784

### CODE REVIEW

File: priv/start-redis-for-test.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#! /bin/sh
```
Comment (zmstone) [inline priv/start-redis-for-test.sh]: please move this script to another dir
`priv` will be packed into releases.

**Comment by zmstone** [comment]
need app up

File: src/eredis.appup.src [hunk 2]
```diff
@@ -40,6 +40,9 @@
     ]},
     {"1.2.5", [
       {load_module, eredis_client, brutal_purge, soft_purge, []}
+    ]},
+    {"1.2.6", [
```
Comment (zmstone) [inline src/eredis.appup.src:44]: Add it to the downgrade section (below) too

**Review by hjianbo — APPROVED** [review]

File: src/eredis.appup.src [hunk 3]
```diff
@@ -40,6 +40,9 @@
...
+      {load_module, eredis_client, brutal_purge, soft_purge, []},
```
Comment (zmstone) [inline src/eredis.appup.src:45]: Syntax error!

File: src/eredis.appup.src [hunk 4]
```diff
@@ -82,6 +85,9 @@
...
```
Comment (zmstone) [inline src/eredis.appup.src:90]: Extra comma in end of the line.

**Comment by zmstone** [comment]
Please Fix syntax err in another pr and force push the tag.

File: src/eredis.appup.src (hunk 3 above)
Comment (gsychev) [reply src/eredis.appup.src:45]:
```suggestion
      {load_module, eredis_client, brutal_purge, soft_purge, []}
```

**Pull request merged**